WebUI/WebUI/settings.py

```diff
@@ -22,9 +22,10 @@
 SECRET_KEY = '2lr1sdqkdl==*-na*%v@ky3k$e8y(qvv5tg)kb!!eu!71ix6y*'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+# https://stackoverflow.com/questions/48276444/django-css-file-not-loading-in-production-debug-false/49398021
```

Comment:
More context on what was fixed is needed.

Reply:
DEBUG = FALSE used to show 404 page error